json_dynamic_widget/example/pubspec.yaml

```diff
@@ -27,9 +27,13 @@ dev_dependencies:
   flutter_test: 
     sdk: 'flutter'
   icons_launcher: '^2.1.6'
-  json_dynamic_widget_codegen: '^1.0.4+4'
+  json_dynamic_widget_codegen: '^1.0.5'
   yaml_writer: '^1.0.3'
 
+dependency_overrides: 
+  json_dynamic_widget_codegen: 
+    path: '../../codegen'
```

Comment:
Is that needed?

Reply:
Is needed to make the version resolving work, since the updated version is not on pub.dev.
The tests don't run if version resolving fails. Since CI already passed once, I could delete it and this be merged even if the new CI will not pass

Reply:
In truth it should probably stay that way.  `dependency_overrides` get ignored when you pull from `pub.dev` and only act on local source.  And you're probably going to want the local codegen because of how closely it's tied to the main package.